app/controllers/tasks_controller.rb

```diff
@@ -26,6 +26,7 @@ def create
   def update
     task.assign_attributes(task_params(task.class))
     task.save!
+    task.send_emails if task.respond_to? :send_emails
```

Comment:
The problem with a no-op'ing `send_emails` is that there is a good chance of implementing send_emails for the Task itself. The Task isn't an abstract class by definition since it contains behavior too. 

WDYT?